app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -40,6 +40,10 @@ def self.default_blacklisted_event_names
     )
   end
 
+  def self.metrics_collect_queue_name
+    MetricsCollectorWorker.default_queue_name
```

Comment:
I'd rather see this pushed up to `ManageIQ::Providers::BaseManager` (and renamed to `metrics_collector_queue_name`).

```ruby
class BaseManager
  def self.metrics_collector_queue_name
    self::MetricsCollectorWorker.default_queue_name
  end
end
```

Then, this is only needed in one place and all of the `CloudManager`s and `InfraManager`s and `OtherManager`s will get it for free.